scalafmt-intellij/src/main/scala/org/scalafmt/intellij/FileDocument.scala

```diff
@@ -17,13 +17,15 @@ case class FileDocument(project: Option[Project], document: Document) {
   def path = virtualFile.getPath
   def isSbt: Boolean =
     virtualFile.getFileType.getName.compareToIgnoreCase("SBT") == 0
+  def isSc: Boolean =
+    virtualFile.getFileType.getName.compareToIgnoreCase("SC") == 0
   def isScala: Boolean = virtualFile.getFileType.getName == "Scala"
   def canFormat: Boolean = isScala || isSbt
 
   def format(): Unit = if (canFormat) {
     val style = getStyle(project, virtualFile)
     val runner =
-      if (isSbt)
+      if (isSbt || isSc)
         style.runner.copy(dialect = scala.meta.dialects.Sbt0137)
```

Comment:
Unrelated to this PR but the line below contains a bug, this line should be
```scala
         style.runner.forSbt
```
to support nice things like trailing commas.
https://github.com/scalameta/scalafmt/blob/542a04500165a8827d2484118526f2c9c3d03835/scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala#L34

Reply:
Done !